openapi_tester/schema_converter.py

```diff
@@ -18,7 +18,9 @@ def __init__(self, schema: dict, with_faker: bool = False):
 
             Faker.seed(0)
             self.faker = Faker()
-        schema_type = schema['type']
+        schema_type = schema.get('type')
+        if not schema_type and 'properties' in schema:
+            schema_type = 'object'
```

Comment:
Isn't there a case where `schema_type` is falsy, and 'properties' won't be in schema? Re the any type discussion

Edit: as in, should we handle the last case in addition to this if?